src/tickit_devices/eiger/eiger_schema.py

```diff
@@ -1,10 +1,18 @@
+import logging
 from dataclasses import dataclass, field
 from enum import Enum
 from functools import partial
 from typing import Any, Generic, List, Mapping, Optional, TypeVar
 
+from apischema import serialized
+from apischema.fields import with_fields_set
+from apischema.metadata import skip
+from apischema.serialization import serialize
```

Comment:
apischema? I'll look into whether this can be pulled out easily

Reply:
Have pulled it out

Reply:
As discussed, pulling this out proved to be more complicated than expected and caused upstream issues with the IOC. It will require a proper look to debug and test those differences. For now I have reset and left this internal use of apischema in. WIP on pulling out here: https://github.com/dls-controls/tickit-devices/pull/66